cmd/main.go

```diff
@@ -250,7 +250,10 @@ func initConfigMapTemplates(ctx context.Context, c client.Client, namespace stri
 		}
 
 		if err := retry.RetryOnConflictOrRetriable(k8sretry.DefaultBackoff, func() error {
-			return client.IgnoreAlreadyExists(c.Create(ctx, configMap))
+			if err := client.IgnoreAlreadyExists(c.Create(ctx, configMap)); err != nil {
+				return fmt.Errorf("failed to create ConfigMap: %w", client.IgnoreAlreadyExists(err))
```

Comment:
I think the second `client.IgnoreAlreadyExists(err)` is redundant here:
```go
return fmt.Errorf("failed to create ConfigMap: %w", client.IgnoreAlreadyExists(err))
```

Reply:
nice catch, thxs!